### ISSUE TITLE
Milestone 1 Feedback

### ISSUE THREAD

**Issue by ahofmann4** [UBC-MDS/dsci-532_group-20#21]
Hi Group 20,

- good job describing your data and user scenario information!
- I like the dashboard design. It is clear, and well described.

- I am wondering if a summary plot to show averages/medians of specific seasons or months next to your main plot would be helpful. For example, I am thinking about what if Mary wants to look at trends for all the months of May, or all summer months? She would not be able to do this with your slider and current graphs proposed.

**Comment by camharris22** [comment]
Thanks for the feedback Analise. We have been discussing the usefulness of the cross plots given our dataset and some alternatives to make the app more suitable for Mary. Stay tuned.

**Comment by trevorki** [comment]
Analise, we have struggled a bit with how to deal with the time on the x-axis. As it stands now we have grouped our data into weekly summaries that are averaged over the years in the dataset. The  user can choose the range of weeks over which to view. We would like to give the option of choosing which year to view data from in  the next milestones, but for now this is what we were able to get working.
